src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -3317,6 +3317,11 @@ private static void ThrowInvalidField(string f)
             result.LastQueryTime = _lastQueryingTime ?? DateTime.MinValue;
             result.ResultEtag = CalculateIndexEtag(queryContext, indexContext, q, result.IsStale);
             result.NodeTag = DocumentDatabase.ServerStore.NodeTag;
+            
+            if (q.SpatialResults != null)
```

Comment:
This means that _every_ time we answer a spatial query we'll be including `SparialProperties` as well. We'd prefer to include it _only_ if it's explicitly requested by the studio. I think we do it this way for time series

Reply:
Added a url param `addSpatialProperties` 
(similar to `addTimeSeriesNames` param)

Reply:
Great!